### PR TITLE
Add hot meal routes

### PR DESCRIPTION
**Pull request by mrbgit**
everything is now inline with JSON best practices for data structure and API design

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,24 +3,36 @@
 
 # Pocket Outreach
 
-This is the REST API for Pocket Outreach, a web app designed to connect people in need with resources in their area.
+This is the REST API for Pocket Outreach, a web app designed to connect people in need with resources in their area. 
+
+The API endpoints and complete documentation are available from [data.pocketoutreach.org](https://data.pocketoutreach.org/)
+
+Currently there are endpoints for locating:
```
Comment (clcuevas) [inline README.md]: There's a typo here.

File: api/foodBanks/lib/getCentralEastNCFoodBanks/getCentralEastNCFoodBanks.test.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    const FoodBankStub = {
+      findOneAndUpdate: (query, obj, options, callback) => {
+        callback(testError);
+      }
+    };
+
+    function errorHandlerStub(error) {
+      expect(error).to.equal(testError);
+    }
+    // eslint-disable-next-line
```
Comment (clcuevas) [inline api/foodBanks/lib/getCentralEastNCFoodBanks/getCentralEastNCFoodBanks.test.js]: Is there a specific ESLint rule you need to disable? If so, please include it in this inline ESLint comment.

File: api/foodBanks/lib/getCentralEastNCFoodBanks/getCentralEastNCFoodBanks.test.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    let getCentralEastNCFoodBanks = proxyquire('./getCentralEastNCFoodBanks', {'../../models/FoodBank': FoodBankStub});
```
Comment (clcuevas) [inline api/foodBanks/lib/getCentralEastNCFoodBanks/getCentralEastNCFoodBanks.test.js]: Let's keep the indentation for curly braces consistent...

```javascript
let getCentralEastNCFoodBanks = proxyquire('./getCentralEastNCFoodBanks', { '../../models/FoodBank': FoodBankStub });
```

File: api/foodBanks/middleware/getFoodBanks/getFoodBanks.test.js [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+        callback(testError);
+      }
+    };
+    const req = {
+      query: {
+        latitude: '47.6795200',
+        longitude: '-122.3875480'
+      }
+    };
+    // eslint-disable-next-line
```
Comment (clcuevas) [inline api/foodBanks/middleware/getFoodBanks/getFoodBanks.test.js]: If your intent is to disable a specific ESLint rule, please include that rule in this inline comment.

File: api/foodBanks/middleware/getFoodBanks/getFoodBanks.test.js [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+    let getHotMeals = proxyquire('./getFoodBanks', {'../../models/FoodBank': foodBankStub});
```
Comment (clcuevas) [inline api/foodBanks/middleware/getFoodBanks/getFoodBanks.test.js]: Please fix the curly braces indentation here. If I recall correctly, your `.eslintrc` file should catch this if the rule is enabled. Can you double check on this, please?

File: api/hotMeals/middleware/getHotMeals/getHotMeals.test.js [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+        callback(testError);
+      }
+    };
+    const req = {
+      query: {
+        latitude: '47.6795200',
+        longitude: '-122.3875480'
+      }
+    };
+    // eslint-disable-next-line
```
Comment (clcuevas) [inline api/hotMeals/middleware/getHotMeals/getHotMeals.test.js]: Ditto here from my previous ESLint feedback.

File: api/hotMeals/middleware/getHotMeals/getHotMeals.test.js [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+    let getClosestHotMeal = proxyquire('./getHotMeals', {'../../models/HotMealLocation': hotMealStub});
```
Comment (clcuevas) [inline api/hotMeals/middleware/getHotMeals/getHotMeals.test.js]: Indentation needed here.

File: README.md (hunk 1 above)
Comment (mrbgit) [inline README.md]: where?

File: api/foodBanks/lib/getCentralEastNCFoodBanks/getCentralEastNCFoodBanks.test.js (hunk 2 above)
Comment (mrbgit) [inline api/foodBanks/lib/getCentralEastNCFoodBanks/getCentralEastNCFoodBanks.test.js]: OK done, I've started doing that but there are old ones to fix

File: api/foodBanks/lib/getCentralEastNCFoodBanks/getCentralEastNCFoodBanks.test.js (hunk 3 above)
Comment (mrbgit) [inline api/foodBanks/lib/getCentralEastNCFoodBanks/getCentralEastNCFoodBanks.test.js]: done

File: api/foodBanks/middleware/getFoodBanks/getFoodBanks.test.js (hunk 4 above)
Comment (mrbgit) [inline api/foodBanks/middleware/getFoodBanks/getFoodBanks.test.js]: done

File: api/foodBanks/middleware/getFoodBanks/getFoodBanks.test.js (hunk 5 above)
Comment (mrbgit) [inline api/foodBanks/middleware/getFoodBanks/getFoodBanks.test.js]: it does, but eslint only looks at touched files so there has to be a change before eslint sees the error, it's fixed now

File: api/hotMeals/middleware/getHotMeals/getHotMeals.test.js (hunk 6 above)
Comment (mrbgit) [inline api/hotMeals/middleware/getHotMeals/getHotMeals.test.js]: :heavy_check_mark:

**Pull request merged**